dev/managedservicesplatform/internal/resource/loadbalancer/loadbalancer.go

```diff
@@ -160,3 +180,50 @@ func New(scope constructs.Construct, id resourceid.ID, config Config) (*Output,
 		ExternalAddress: externalAddress,
 	}, nil
 }
+
+// cloudArmorAllowOnlyCloudflareEdge defines a security policy which only allows traffic from Cloudflare edge IPs
+func cloudArmorAllowOnlyCloudflareEdge(scope constructs.Construct, id resourceid.ID, config Config) *string {
+	securityPolicy := computesecuritypolicy.NewComputeSecurityPolicy(scope, id.TerraformID("security-policy"), &computesecuritypolicy.ComputeSecurityPolicyConfig{
+		Name:    pointers.Stringf("%s-security-policy", *config.TargetService.Name()),
+		Project: pointers.Ptr(config.ProjectID),
+		Rule: []computesecuritypolicy.ComputeSecurityPolicyRule{
+			{
+				Description: pointers.Ptr("Deny All"),
+				Action:      pointers.Ptr("deny(403)"),
+				Priority:    pointers.Ptr(2147483647.0),
```

Comment:
```suggestion
				Priority:    pointers.Ptr(2147483647.0), // arbitrarily high value for ...
```

presumably to make sure this has high precedence?

Reply:
The opposite!

Rules are evaluated from highest priority (lowest numerically) to lowest priority (highest numerically) in order

It is also necessary to have a default rule with priority 2147483647 (not arbitrary) and CIDR match `*` to handle all traffic not matched by higher priority rules.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_security_policy#rule

Reply:
> It is also necessary to have a default rule with priority 2147483647 (not arbitrary) and CIDR match * to handle all traffic not matched by higher priority rules.

Thanks, TIL!!! this is all super helpful context, let's get this added in the docstrings here 😁